frontend/src/components/CustomerWeightsVisualization.tsx

```diff
@@ -0,0 +1,152 @@
+import { FC, useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { skipToken } from '@reduxjs/toolkit/query/react';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import Tooltip from '@mui/material/Tooltip';
+import { chosenRoadmapIdSelector } from '../redux/roadmaps/selectors';
+import { CustomerStakes } from '../redux/roadmaps/types';
+import css from './CustomerWeightsVisualization.module.scss';
+import { apiV2 } from '../api/api';
+import { Dot } from './Dot';
+import { percent } from '../utils/string';
+import { PercentageBar } from './PercentageBar';
+
+const classes = classNames.bind(css);
+
+export const CustomerWeightsTooltipContent: FC<{
+  customerStakes: CustomerStakes[];
+  totalValue: number;
+  showPercentageBar?: boolean;
+  noTitle?: true;
+}> = ({ customerStakes, totalValue, showPercentageBar, noTitle }) => {
+  const { t } = useTranslation();
+
+  if (customerStakes.length === 0)
+    return (
+      <div className={classes(css.stakesTooltip)}>
+        {t('No client weights set')}
+      </div>
+    );
+
+  return (
+    <div className={classes(css.stakesTooltip)}>
+      {!noTitle && <div>{t('Client weights')}</div>}
+      {showPercentageBar && (
+        <PercentageBar
+          stakes={customerStakes}
+          totalValue={totalValue}
+          width={15}
+        />
+      )}
+      {customerStakes.map((entry) => (
+        <div
+          key={entry.id}
+          className={classes(css.customer, {
+            [css.differsTooMuch]: entry.differsTooMuchFromPlanned,
+          })}
+        >
+          <Dot fill={entry.color} />
+          <span className={classes(css.name)}>{entry.name}</span>
+          <span className={classes(css.percentage)}>
+            {percent(1).format(totalValue > 0 ? entry.value / totalValue : 0)}
+          </span>
+        </div>
+      ))}
+    </div>
+  );
+};
+
+const CustomerWeightsPercentageBar: FC<{
+  customerStakes: CustomerStakes[];
```

Comment:
`filter: opacity(70%)` style for the percentage bar in this component should work for the different look (the 70% is what was in figma)